components/src/preact/lineageFilter/lineage-filter.tsx

```diff
@@ -118,7 +121,10 @@ const LineageFilterInner: FunctionComponent<LineageFilterInnerProps> = ({
             </div>
             <datalist id={lapisField}>
                 {data.map((item) => (
-                    <option value={item} key={item} />
+                    <option
+                        value={item[lapisField]}
+                        key={item[lapisField]}
+                    >{`${item[lapisField]} (${item['count']})`}</option>
```

Comment:
You don't need a template string here. You can use JSX string rendering directly.
```suggestion
                    >{item[lapisField]} ({item['count']})</option>
```